### PR TITLE
📖UPDATE DOC: Fix grammar and make doc precise

### PR DESCRIPTION
**Pull request by saqibameen**
Update Heading: It's obvious that's it's an introduction to Volto. So, `to Volto` is excessive in the heading.
Minor Grammar Fixes: A few minor grammar fixes are introduced to remove `fluff`. Resulting in a more objective and precise document.

### CODE REVIEW

**Comment by saqibameen** [comment]
@sneridagh can you help me guide through the process of contributing to the docs?

File: docs/source/00-introduction/introduction.md [hunk 1]
```diff
@@ -1,33 +1,13 @@
...
 
-Plone is a CMS built on Python born back in 2001, and unmatched experience in
-the subject.
+Plone is an CMS, which was written in Python back in 2001 and cherishes an unmatched experience in the subject. It has exciting features that are still appealing to developers and users alike. Such as customizable content types, hierarchical URL object traversing, and a sophisticated content workflow powered by a granular permissions model that allows you to build from simple websites to huge complex intranets.
 
-Plone has very interesting features that are still appealing to developers and
-users alike as customizable content types, hierarchical URL object traversing
-and a complex content workflow powered by a granular permissions model that
-allows you to build from simple websites to complex huge intranets.
+Volto exposes all those features and communicates with Plone via its mature REST API. It can be highly themeable and customizable. Volto also supports other APIs like Guillotina, a Python resource management system, which is inspired on Plone using the same basic concepts like traversal, content types, and permissions model.
```
Comment (tisto) [inline docs/source/00-introduction/introduction.md]: Isn't it "inspired by Plone" instead of "inspired on Plone"?

Comment (saqibameen) [reply docs/source/00-introduction/introduction.md]: @tisto ops! yes, you are right. just fixed it.

**Comment by sneridagh** [comment]
@saqibameen @tisto Sorry to jump in so late. This piece of documentation is already corrected and validated by @polyester during the Sorrento Sprint. I forgot to update it in the docs :(

The corrected and already reviewed text is in the https://github.com/plone/volto/blob/master/README.md

So if you want to make an amendment, you should start using that one as template.

Thanks and sorry again for all the hassle.

**Comment by saqibameen** [comment]
My bad @sneridagh, I spent time amending the docs. However, it's great that they have already been updated.

Actually, I got to know that [Volto Docs](https://docs.voltocms.com/) require contributions through GSoD. Which are not updated yet. I am interested in contributing to these docs but apparently, there's no slack channel/ forum where I can get connected with mentors and get started. 

Any pointers on where can I reach out mentors and get started with it? 

Thanks,

**Comment by sneridagh** [comment]
@saqibameen on the contrary, my bad for not jump in before. If you are interested in the GSoD, please follow the official channels to apply as an official candidate. There are people at the Plone Foundation that are already taken care of it. Good luck and hope to see you made it!

**Comment by saqibameen** [comment]
@sneridagh thanks for your response. I was actually trying to reach out the foundation to discuss the projects, but unfortunately, there's no channel provided for this purpose. The only email given is to send the proposal. 

However, thanks for wishing me luck! :)

**Comment by sneridagh** [comment]
@saqibameen I can only say this: apply as candidate, send the proposal with it (don't know if that's possible but try). It will be heard and answered for sure. However, the projects are the ones announced and the reach is already set by the foundation.

**Pull request closed without merging**